### ISSUE TITLE
uneven color on the external screen

### ISSUE THREAD

**Issue by Jeffrey-Lcc** [atom/atom#14807]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * [X[Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * [X}Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * [X]Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * [X]Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * [X]Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

There is uneven color on the screen on the external monitor on the current version, but not when setting tab is on or on the atom beta version 1.90 
### Steps to Reproduce

1. simply open the atom app
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expect to happen]

**Actual behavior:** [What actually happens]

**Reproduces how often:** [What percentage of the time does it reproduce?]
100%

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.

Atom    : 1.18.0
Electron: 1.3.15
Chrome  : 52.0.2743.82
Node    : 6.5.0

apm  1.18.1
npm  3.10.10
node 6.9.5 x64
python 2.7.10
git 2.10.1
![screenshot_16_6_2017__19_11](https://user-images.githubusercontent.com/25099352/27224947-72f6203e-52ca-11e7-9cf9-db0d7d269a5d.png)

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.
![screenshot_16_6_2017__19_33](https://user-images.githubusercontent.com/25099352/27224998-c02fecd6-52ca-11e7-9f9c-17a5432578d7.png)

**Comment by BlaM** [comment]
What do you mean with "external screen"?

**Comment by pardom** [comment]
I've been seeing this for a while, too. It only happens on an external display. You can reproduce by moving the window between the built-in display and the external display.


**macOS** Sierra
Version 10.12.5

**MacBook Pro (15-inch, 2016)**
**Processor** 2.6 GHz Intel Core i7
**Memory** 16 GB 2133 MHz LPDDR3
**Graphics** Radeon Pro 450 2048 MB / Intel HD Graphics 530 1536 MB

```shell
$ atom --version
Atom    : 1.18.0
Electron: 1.3.15
Chrome  : 52.0.2743.82
Node    : 6.5.0
```

```shell
$ apm --version
apm  1.18.1
npm  3.10.10
node 6.9.5 x64
python 2.7.13
git 2.13.1
```

```shell
$ atom --list
Community Packages (23) /Users/michael/.atom/packages
├── Sublime-Style-Column-Selection@1.7.4
├── atom-beautify@0.29.26
├── autocomplete-haskell@0.7.2
├── busy-signal@1.4.3
├── file-icons@2.1.7
├── haskell-debug@0.2.2
├── haskell-ghc-mod@1.19.9
├── haskell-pointfree@0.2.0
├── ide-haskell@1.9.6
├── ide-haskell-cabal@1.9.2
├── ide-haskell-hasktags@0.0.11
├── ide-haskell-hoogle@0.0.6
├── intentions@1.1.2
├── language-elm@1.5.0
├── language-haskell@1.12.1
├── language-haskell-scoped@0.0.4
├── language-kotlin@0.5.0
├── linter@2.1.4
├── linter-kotlin@0.5.0
├── linter-ui-default@1.6.1
├── my-theme-syntax@0.0.0
├── script@3.15.0
└── set-syntax@0.3.2
```

![screen shot 2017-06-16 at 9 12 42 am](https://user-images.githubusercontent.com/189503/27228031-fdc5ac72-5273-11e7-8549-28cecf95b97d.png)

**Comment by ryanluton** [comment]
running into the same issue. Have tried multiple themes, window sizes, and external monitors. 
<img width="1148" alt="screenshot 2017-06-16 08 26 01" src="https://user-images.githubusercontent.com/3237400/27228538-a5992cb4-526d-11e7-89ea-88f75ecd5c3f.png">

**Comment by 50Wliu** [comment]
This has been fixed in Atom 1.19.0-beta0.
